### ISSUE TITLE
[BUG] cudf Series string output doesn't match pandas

### ISSUE THREAD

**Issue by thomcom** [rapidsai/cudf#740]
**Describe the bug**
Pandas Series output includes the name of the series and the dtype. I accept this as a feature request, not a bug, except for someone who wants to parse pandas output as a stream. Good first issue.

**Steps/Code to reproduce bug**
```
x = pd.Series([1,2,3], name='a')
y = cudf.Series([1,2,3], name='a')
print(x)
print(y)
```
output:
```
0    1
1    2
2    3
Name: a, dtype: int64
      
0    1
1    2
2    3
```
**Expected behavior**
We should include `Name: <name>, dtype: <dtype>` when we convert a `Series` to string.

**Comment by thomcom** [comment]
Empty series also produces a considerably different output:

```
z = pd.Series([], name="e")
Z = cudf.Series([], name="e")
print(z.sort_values())
print(Z.sort_values())
```
output:
```
Series([], Name: e, dtype: float64)
<empty Series of dtype=float64>
```

**Comment by kkraus14** [comment]
Resolved by #878
